src/migrations/20210217195834-rbac-tables.js

```diff
@@ -0,0 +1,81 @@
+exports.up = function(db, cb) {
+    db.runSql(
+        `CREATE TABLE IF NOT EXISTS roles
+       (
+          id          SERIAL PRIMARY KEY,
+          name        text not null,
+          description text,
+          type        text not null default 'custom',
+          project     text,
+          created_at  TIMESTAMP WITH TIME ZONE DEFAULT now()
+       );
+      CREATE TABLE IF NOT EXISTS role_user
+      (
+          role_id     integer not null references roles (id) ON DELETE CASCADE,
+          user_id     integer not null references users (id) ON DELETE CASCADE,
+          created_at  TIMESTAMP WITH TIME ZONE DEFAULT now(),
+          PRIMARY KEY (role_id, user_id)
+      );
+      CREATE TABLE IF NOT EXISTS role_permission
+      (
+          role_id     integer not null references roles (id) ON DELETE CASCADE,
+          project     text,
+          permission  text not null,
+          created_at  TIMESTAMP WITH TIME ZONE DEFAULT now()
+      );
+    
+      WITH admin AS (
+        INSERT INTO roles(name, description, type)
+        VALUES ('Admin', 'Can read and write all resources in Unleash. This includes add and remove users and roles.', 'root')
```

Comment:
@FredrikOseberg I would love your feedback on these Role descriptions. They will typically be shown in the UI as helpers.